packages/sockjs-shim/README.md

```diff
@@ -0,0 +1,9 @@
+# `sockjs-shim`
+[Source code of released version](https://github.com/meteor/meteor/tree/master/packages/cockjs-shim) | [Source code of development version](https://github.com/meteor/meteor/tree/devel/packages/sockjs-shim)
```

Comment:
I believe there's a typo in the first link here

Reply:
oh dear